pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/vedlegg/omstillingsstoenad/BeregningAvOmstillingsstoenad.kt

```diff
@@ -190,87 +193,130 @@ private fun OutlineOnlyScope<LangBokmalNynorskEnglish, OmstillingsstoenadBeregni
-                        "assumed that your estimated remaining annual income will be NOK " + inntekt.format() +
-                        " from " + virkningsdato.format() + ".",
+                Bokmal to "Din oppgitte årsinntekt for inneværende år er ".expr() + aarsinntekt.format() + " kroner. Fratrekk " +
+                    "for inntekt i måneder du ikke er innvilget stønad er " + fratrekkInnAar.format() + " kroner. Vi har lagt til grunn " +
+                    "at du har en inntekt på " + inntekt.format() +
+                    " kroner i innvilgede måneder i år. Beløpet er avrundet ned til nærmeste tusen",
+                Nynorsk to "Du har ei oppgitt årsinntekt inneverande år på ".expr() + aarsinntekt.format() + " kroner. " +
+                    "Fråtrekk for inntekt i månader du ikkje er innvilga stønad er " + fratrekkInnAar.format() +
+                    " kroner. Vi har lagt til grunn " +
+                    "at du har ei inntekt på " + inntekt.format() +
+                    " kroner i innvilga månader i år. Beløpet er avrunda ned til næraste tusen.",
+                English to
+                    "Your estimated annual income for the current year is NOK ".expr() + aarsinntekt.format() + ". " +
+                    "Deduction for income in months you are not granted allowance is NOK " + fratrekkInnAar.format() + ". " +
+                    "We have assumed that you have an income of NOK " + inntekt.format() + " in granted months this year." +
+                    "The amount is rounded down to the nearest thousand.",
             )
         }
         paragraph {
             textExpr(
                 Bokmal to "Stønad per måned er redusert på følgende måte: (".expr() + inntekt.format() +
+                    " kroner / " + gjenvaerendeMaaneder.format() + " måneder) minus (0,5 G / 12 måneder). " +
+                    "Beløpet ganges med 45 prosent.",
+                Nynorsk to "Stønaden per månad har blitt redusert på følgjande måte: (".expr() + inntekt.format() +
+                    " kroner / " + gjenvaerendeMaaneder.format() + " månader) minus (0,5 G / 12 månader). " +
+                    "Beløpet blir gonga med 45 prosent.",
+                English to "The monthly allowance amount has been reduced as follows: (NOK ".expr() +
+                    inntekt.format() + " / " + gjenvaerendeMaaneder.format() + " months) minus (0.5 G / 12 months). " +
+                    "The amount is multiplied by 45 percent.",
+            )
+        }
+
+        showIf(harInntektForNesteAar) {
+            val januarNesteAar =
+                YearMonth
+                    .now()
+                    .plusYears(1)
+                    .withMonth(1)
+                    .atDay(1)
+                    .format(DateTimeFormatter.ofPattern("dd.MM.yyyy"))
+
+            // TODO: skal denne bruke ny oppgitt inntekt eller samme som over?
```

Comment:
ah right dette blir jo feil. Eller det blir riktig her men feil over. Nå vil jo inntekt for neste år vises begge steder.
Det må legges til enda et felt "sisteBeregningsperiodeNesteAar" slik at det kan hentes inntekt og inntektNesteAar.
Kan utledes på Gjenny-siden slik
`
            val sisteBeregningsperiode =
                beregningsperioder.filter { it.datoFOM.year == beregningsperioder.first().datoFOM.year }
                    .maxBy { it.datoFOM }
            val sisteBeregningsperiodeNesteAar =
                beregningsperioder.filter { it.datoFOM.year == beregningsperioder.first().datoFOM.year + 1 }
                    .maxBy { it.datoFOM }
`